app/views/docs/realtime.phtml

```diff
@@ -118,8 +118,8 @@ let subscription = realtime.subscribe(channels: ["files"]) { message in
 
 <div class="notice margin-top-large margin-bottom-large">
     <h3>Permissions</h3>
-    <p>All subscriptions are secured by the <a href="/docs/permissions">permissions</a> offered by Appwrite, meaning a user will only receive updates to resources to which he has read permission.</p>
-    <p>Using <span class="tag">role:all</span> on read permissions will allow any client to receive updates.</p>
+    <p>All subscriptions are secured by the <a href="/docs/permissions">permissions system</a> offered by Appwrite, meaning a user will only receive updates to resources to which he has permission to read.</p>
+    <p>Using <span class="tag">read(all)</span> on read permissions will allow any client to receive updates.</p>
```

Comment:
Should we do SDK syntax here, `Permission.read(Role.any())`?

Even if not, this should probably be `any` instead of `all`.

Reply:
Let's do SDK syntax everywhere because Magic Strings are evil.